stack.yaml

```diff
@@ -1,74 +1,11 @@
-# This file was automatically generated by 'stack init'
```

Comment:
interesting. but how did you update the `extra-deps` section? I thought you manually edit it?